stage1/rootfs/usr_from_kvm/kernel/patches/reboot.patch

```diff
@@ -0,0 +1,22 @@
+--- linux-4.0/kernel/reboot.c~	2015-04-12 18:12:50.000000000 -0400
++++ linux-4.0/kernel/reboot.c	2015-05-06 09:45:56.991894271 -0400
+@@ -309,7 +309,7 @@
+ 	 * halt when pm_power_off is not set do it the easy way.
+ 	 */
+ 	if ((cmd == LINUX_REBOOT_CMD_POWER_OFF) && !pm_power_off)
+-		cmd = LINUX_REBOOT_CMD_HALT;
++		cmd = LINUX_REBOOT_CMD_RESTART;
```

Comment:
I wonder if some systemd units like `poweroff.target` or `systemd-reboot.service` could be changed so that systemd requests a reboot instead of a halt. That would achieve the same goal without patching the kernel.

Some of those units are being introduced by #1143.